### PR TITLE
added diagnose to torso

### PR DESCRIPTION
**Pull request by alganthe**
added the interaction as it was missing but still marked in the changelog

### CODE REVIEW

**Comment by PabstMirror** [comment]
Should part be `body` instead of `head`?

**Comment by alganthe** [comment]
it's part of  `torso` not `head`

File: addons/medical/ACE_Medical_Actions.hpp [hunk 1]
```diff
@@ -122,6 +122,17 @@ class ACE_Torso {
         enableInside = 1;
         icon = PATHTOF(UI\icons\triageCard.paa);
     };
+    class Diagnose {
+        displayName = CSTRING(Actions_Diagnose);
+        distance = 5.0;
+        condition = QUOTE([ARR_4(_player, _target, 'head', 'Diagnose')] call DFUNC(canTreatCached));
```
Comment (Glowbal) [inline addons/medical/ACE_Medical_Actions.hpp]: This should be `body`, not `head`

**Comment by Glowbal** [comment]
Merged into release with #2557

**Pull request closed without merging**